x/multisig/codec.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package multisig;
+
+message Contract {
+    // contract's address
+    bytes address = 1;
```

Comment:
The address is stored as the primary key, no need to store it in the value.
Otherwise, the object looks good.

I would actually suggest using an orm.Sequence to auto-generate an auto-incrementing integer for each Contract, which is the primary key (I think sequence exports int64 or [8]byte)